index.js

```diff
@@ -13,71 +7,12 @@ function ask(questionText) {
   });
 }
 
-function findMiddlePoint(min, max) {
-  let middle = Math.floor((min + max) / 2)
-  return middle;
-}
-
-
 start();
 
 async function start() {
-
-  console.log("Let's play a game where you (human) make up a number between 1 and 100 and I (computer) try to guess it.")
+  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-  let guess = await ask("is it " + guessnum + "?\n");
-
-
-
-
-  if (guess.toLowerCase() === 'n' || guess.toLowerCase() === 'no') {
-
-    guessing = true;
-    while (guessing === true) {
-      let highLow = await ask('Is it higher (H), or lower (L)?');
-      if (highLow.toUpperCase() === 'H') {
-        min = guessnum + 1
-        guessnum = findMiddlePoint(min, max);
-        guess = await ask("is it " + guessnum + "?\n");
-        guessCount += 1
-
-        if (guess.toLowerCase() === 'y' || guess.toLowerCase() === 'yes') {
-          guessing = false
-        }
-      } else if (highLow.toUpperCase() === 'L'){
-        max = guessnum - 1
-        guessnum = findMiddlePoint(min, max);
-        guess = await ask("is it " + guessnum + "?\n");
-        guessCount += 1
-        if (guess.toLowerCase() === 'y' || guess.toLowerCase() === 'yes') {
-          guessing = false
-        }
```

Comment:
You could put this (and the if check from above) outside of this logic block but still inside the while loop (between lines 59 and 60), and then you would only have to put it in your code once.